modules/platforms/cpp/ignite/odbc/type_traits.cpp

```diff
@@ -596,8 +596,36 @@ std::int32_t ignite_type_num_precision_radix(ignite_type typ) {
     return sql_type_num_precision_radix(sql_type);
 }
 
-std::int32_t sql_type_decimal_digits(std::int16_t) {
-    // TODO: IGNITE-19854 Remove once parameters meta fetching is implemented
+std::int32_t sql_type_decimal_digits(std::int16_t type) {
```

Comment:
Should we rename this method? Currently it is confusing: for `DECIMAL` we return `-1`.

Reply:
I decided to add a `scale` parameter for this function instead. It's name confusing, but it's [ODBC terminology](https://learn.microsoft.com/en-us/sql/odbc/reference/appendixes/column-size-decimal-digits-transfer-octet-length-and-display-size), it's really very confusing in this aspect.